scripts/observers/config.py

```diff
@@ -32,6 +32,17 @@ def update_toml_node(path, shard_id):
     toml.dump(prefs_data, f)
     f.close()
 
+    # config.toml
+    num_of_shards = int(os.getenv('NUM_OF_SHARDS'))
+    path_config = path / "config.toml"
+    config_data = toml.load(path_config)
+    config_data['DbLookupExtensions']['Enabled'] = True
+    config_data['EpochStartConfig']['RoundsPerEpoch'] = 20
```

Comment:
[comment]

From experience on other repos, this might occasionally fail on github actions.
So **gj** for thinking of moving it to TC

Reply:
yes. will made a script in the next PR